dask_kubernetes/classic/kubecluster.py

```diff
@@ -626,6 +629,9 @@ async def _start(self):
 
         await super()._start()
 
+        port = await port_forward_dashboard(self.name, self.namespace)
+        self.scheduler_info["services"]["dashboard"] = port
```

Comment:
@jacobtomlinson - Do you think its okay to mutate `scheduler_info` here? It was the only way I found to update `dashboard_link` with the way its currently written in [distributed](https://github.com/dask/distributed/blob/main/distributed/deploy/cluster.py#L323).